lib/constants.dart

```diff
@@ -33,8 +33,14 @@ final kLocalProjectPubspec =
 
 /// FVM Home directory
 String get fvmHome {
-  var home = '';
   final envVars = Platform.environment;
+
+  final fvmHome = envVars['FVM_HOME'];
```

Comment:
Thanks for the PR!

Just a small change which is not to use `fvmHome` since it's the name of the getter already. You can use the `home` and assign to it.

Reply:
Actually no need. I had to make a change for the dart scoring so I am just merging. Thanks again for the PR!